state/requests.go

```diff
@@ -45,11 +45,12 @@ type DeleteStateOption struct {
 
 // SetRequest is the object describing an upsert request.
 type SetRequest struct {
-	Key      string            `json:"key"`
-	Value    interface{}       `json:"value"`
-	ETag     *string           `json:"etag,omitempty"`
-	Metadata map[string]string `json:"metadata,omitempty"`
-	Options  SetStateOption    `json:"options,omitempty"`
+	Key         string            `json:"key"`
+	Value       interface{}       `json:"value"`
+	ETag        *string           `json:"etag,omitempty"`
+	Metadata    map[string]string `json:"metadata,omitempty"`
+	Options     SetStateOption    `json:"options,omitempty"`
+	ContentType string            `json:"contentType,omitempty"`
```

Comment:
Please, use metadata for now. We need to discuss ContentType for all APIs and not only this one.

Reply:
Hi @artursouza will take this up in state-management channel in discord.